packages/web/src/pages/Modals.tsx

```diff
@@ -25,11 +25,15 @@ import RewardsModals from 'pages/audio-rewards-page/components/modals/RewardsMod
 import { getClient } from 'utils/clientUtil'
 
 const NATIVE_MOBILE = process.env.REACT_APP_NATIVE_MOBILE
+const NATIVE_NAVIGATION_ENABLED =
+  process.env.REACT_APP_NATIVE_NAVIGATION_ENABLED === 'true'
 
 const Modals = () => {
   const client = getClient()
   const isMobileClient = client === Client.MOBILE
 
+  if (NATIVE_NAVIGATION_ENABLED) return null
```

Comment:
This could be risky right? If a modal has data fetching effects in it for example

Reply:
yeah thats what im wondering as well... cause none of the modals have an actual route associated so i thought maybe wed be fine. but yeah maybe we just see if suddenly we are missing important data on the native modals?

Reply:
Yeah let's keep an eye on it, should be caught in qa if there is an issue